README.md

```diff
@@ -74,88 +75,88 @@ Thanks to those who have [helped to make Metrodroid available in languages other
 
 Card / Agency | Location | Notes
 ------------- | -------- | -----
-[Amiibo][amiibo] | :earth_asia: _worldwide_ | :new:
-[AT HOP][athop] | :new_zealand: Auckland, New Zealand | :new: :id:
+[Amiibo][amiibo] | :earth_asia: _worldwide_ | :new: `IOS`
+[AT HOP][athop] | :new_zealand: Auckland, New Zealand | :new: :id: `IOS`
 [Beijing Municipal Card][beijing] | :cn: Beijing, China | :new:
 [bip!][bip] | :chile: Santiago de Chile, Chile | :new: :closed_lock_with_key: `MFC`
 [Bilhete Único][bu] | :brazil: São Paulo, Brazil | :closed_lock_with_key: `MFC`
 [BUS-IT][busit] | :new_zealand: Greater Hamilton (Waikato), New Zealand | :new: :closed_lock_with_key: `MFC`
-[Carta Mobile][pisa] | :it: Pisa, Italy | :new:
+[Carta Mobile][pisa] | :it: Pisa, Italy | :new: `IOS`
 [CharlieCard][charlie] | :us: Boston, MA, USA | :new: :closed_lock_with_key: `MFC`
-City Union | :cn: Mainland China | :new:
-[Clipper][clipper] | :us: San Francisco, CA, USA
-[Compass][compass] | :canada: Vancouver, BC, Canada | :new: :one:
+City Union | :cn: Mainland China | :new: `IOS`
+[Clipper][clipper] | :us: San Francisco, CA, USA | `IOS`
+[Compass][compass] | :canada: Vancouver, BC, Canada | :new: :one: `IOS`
 [Crimea Trolleybus][crimea] | Crimea | :new: :closed_lock_with_key: `MFC`
 [Cubic Nextfare][nextfare] | :earth_americas: _many locations_ | :new: :closed_lock_with_key: `MFC`
 [EasyCard][easycard] | Taipei | :closed_lock_with_key: `MFC`
 [Edy][edy] | :jp: Japan
```

Comment:
I don't have an Edy card to test, so I didn't mark as "iOS supported".  Edy was bought out by Rakuten in 2012, so the functionality may have changed since this got into Farebot.  Our current implementation is FeliCa based; so should work (provided we don't hit issue #613).

It is different to IC/Suica, as it is a pre-paid debit card first, rather than a transit card that also allows retail purchases.

[It looks like you can only purchase this in Japan](https://item.rakuten.co.jp/edyshop/edy-rpoint_red/), but it can be topped up with cash so should be possible to use without a Japanese bank account.